src/MongoDB.Driver.Core/Core/Operations/ChangeStreamCursor.cs

```diff
@@ -154,12 +199,54 @@ private IEnumerable<TDocument> DeserializeDocuments(IEnumerable<RawBsonDocument>
 
             if (lastRawDocument != null)
             {
-                _changeStreamOperation.DocumentResumeToken = lastRawDocument["_id"].DeepClone().AsBsonDocument;
+                _documentResumeToken = lastRawDocument["_id"].DeepClone().AsBsonDocument;
             }
 
             return documents;
         }
 
+        private ResumeStartValue GetEffectiveResumeStartValues()
```

Comment:
Rename to GetResumeValues.

"Effective" made sense when this method was ChangeStreamOperation, but here in this class the shorter name is better.

Reply:
Done